assemblies/rosa-private-cluster.adoc

```diff
@@ -0,0 +1,21 @@
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="private-cluster"]
+:context: private-cluster
+
+= Configuring a private cluster
+
+[role="_abstract"]
+An {product-title} cluster can be made private so that internal applications can be hosted inside a corporate network. In addition, private clusters can be configured to have only internal API endpoints for increased security.
+
+// {product-title} administrators can choose between public and private cluster configuration from within the *OpenShift Cluster Manager* (OCM).
+
```

Comment:
or using the rosa CLI.

Reply:
That paragraph is commented out for this release.